### ISSUE TITLE
How to Build/Compile

### ISSUE THREAD

**Issue by TheDude53** [project64/project64#1697]
I would like to attempt to build Project64 myself and am not sure where to start. I have the latest version of VS Code. Any help is appreciated.

**Comment by flagrama** [comment]
You'll almost certainly need Visual Studio Community 2015 (or later, but that may require further work)
Even if it is possible with Visual Studio Code it would likely require a lot more setup that just installing VS2015 would do for you, such as the msvc libraries and headers, the Windows SDK, etc.

**Comment by TheDude53** [comment]
But it is possible with VS Code?

**Comment by flagrama** [comment]
It is very likely possible, but would require advanced knowledge of how the actual Visual Studio IDE works so that you can translate what it would do to VS code and the command line tools the full version of Visual Studio uses in the background.

**Comment by Mhowser** [comment]
Can we use https://vscodium.com?

**Comment by flagrama** [comment]
That is still just VSCode. Everything that applies to VSCode applies to that too.

**Comment by flagrama** [comment]
Maybe you guys don't realize that Visual Studio Community is free? You can't legally sell applications you make with it, but you are allowed to develop and release personal and open source projects you make or modify with it so long as you are not selling them commercially.

Just download VS2015 Community and open the solution. I assume most, if not all of building Project64 will be relatively simple from there.

**Comment by Sebanisu** [comment]
I think you can make money. I just thought you couldn't use without paying based on type of work and size of your organization.

**Comment by flagrama** [comment]
You are correct, I was mistaken. I've struck out the incorrect parts in the previous comment.

**Comment by JunielKatarn** [comment]
> I would like to attempt to build Project64 myself and am not sure where to start. I have the latest version of VS Code. Any help is appreciated.

You need VS compilers and Windows SDK.
Install Visual Studio 2019.
Enable the Desktop C++ workflow.

You can build it opening the VS solution, or using `MSBuild`.
